### PR TITLE
fix: build doesn't work if inputsRoot dir includes dead symlink

### PR DESCRIPTION
**Pull request by shisama**
`duck serve` doesn't work with dead symlink. That's why [recursive-readdir](https://github.com/jergason/recursive-readdir) occurs an error when 'inputsRoot' directory includes some dead symlinks.
There is a [PR](https://github.com/jergason/recursive-readdir/pull/60/files) to fix it. 
However, recursive-readdir is not active very much. The PR was created over a year ago.
So, I recommend to use [glob](https://github.com/isaacs/node-glob) instead of recursive-readdir.

### CODE REVIEW

File: src/gendeps.ts [hunk 1]
```diff
@@ -99,7 +100,7 @@ export async function getDependencies(
       if (entryConfig["test-excludes"]) {
         testExcludes = entryConfig["test-excludes"];
       }
-      const files = await recursive(p, ignoreDirPatterns);
+      const files = await globPromise(path.join(p, "**/*.js"), { ignore: ignoreDirPatterns });
       return Promise.all(
         files
           .filter(file => /\.js$/.test(file))
```
Comment (teppeis) [inline src/gendeps.ts]: This filter is no longer needed.
```suggestion
```

**Comment by shisama** [comment]
@teppeis Thanks for your review. I fixed them. Would you please review this again?

**Comment by teppeis** [comment]
@shisama thanks!

**Comment by teppeis** [comment]
🚀 Released  https://github.com/teppeis/duck/releases/tag/v0.29.2

**Pull request merged**